### ISSUE TITLE
RFT plotter: fails if I deselect all ensemble under Formation Plot Settings

### ISSUE THREAD

**Issue by lindjoha** [equinor/webviz-subsurface#835]
If I deselect all ensembles under Formation Plot Settings in the RFT plotter (RFT map tab), then I get this error. And I'm not able to get back in without restarting the app.

```
  File "/prog/res/komodo/2021.10.04-py36-rhel7/root/lib/python3.6/site-packages/webviz_subsurface/plugins/_rft_plotter/_formation_figure.py", line 201, in add_ert_observed
    "REAL": df["REAL"].unique()[0],
IndexError: index 0 is out of bounds for axis 0 with size 0
```

**Comment by anders-kiaer** [comment]
Good catch. I observe the same on https://webviz-subsurface-example.azurewebsites.net/rft-plotter if I remove all ensembles there.

The easiest would have been `dropdown` + `multi` + not cleerable (i.e. you need at least one selected value). But that is maybe not possible with `dcc.Dropdown` 🤔 Suggestions on what would be an easy solution here @tnatt @HansKallekleiv? Maybe the easiest to allow for no ensembles and then no plots? That is the current behavior in https://webviz-subsurface-example.azurewebsites.net/simulation-time-series.

**Comment by anders-kiaer** [comment]
The same problem seems to be there with:
* https://webviz-subsurface-example.azurewebsites.net/regional-simulation-time-series
* https://webviz-subsurface-example.azurewebsites.net/pvt
* https://webviz-subsurface-example.azurewebsites.net/parameter-parallel-coordinates (this one uses `wcc.Select` and is therefore a bit more difficult for the user to actually get zero ensembles).
@asnyv @rubenthoms

The other plugins seems to gracefully plot nothing (if no ensembles are selected).

**Comment by HansKallekleiv** [comment]
no way to prevent this behavior afaik. The simplest solution is to catch it first thing in the callback and return a `dash.no_update`. Alternatively we could return an empty figure with some descriptive text as done in some plugins.

**Comment by lindjoha** [comment]
This is easily fixed in the RftPlotter by adding this to the callback:

```
        if not ensembles:
            return {"layout": {"title": "No ensembles selected"}}
```
